source/documentation/troubleshooting/ssh.md

```diff
@@ -8,15 +8,23 @@ If you do run commands which will change the container temporarily, it's a good
 
 SSH is enabled by default. In most cases, you will find that you can SSH directly to your app's container.
 
-Simply run:
+1. Run:
 
-```
-cf ssh APPNAME
-```
+    ```
+    cf ssh APPNAME
+    ```
+
+    where `APPNAME` is the name of the app. 
+
+2. For some tasks to work, you need to set up the interactive SSH session to match the buildpack environment. To do this, run:
 
-where `APPNAME` is the name of the app. 
+    ```
+    export HOME=/home/vcap/app; [ -d ~/.profile.d ] && for f in ~/.profile.d/*; do source $f; done
```

Comment:
This doesn't quite match what's specified in the [upstream docs](https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#ssh-env). It's missing
```sh
source /home/vcap/app/.profile
```